src/Views/Filter/Subject.cshtml

```diff
@@ -44,7 +44,38 @@
                       @foreach (var subject in area.Subjects.OrderBy(x => x.Name)) {
                         <div class="govuk-checkboxes__item">
                           <input id="@("subject- " + subject.Id)" name="SelectedSubjects" type="checkbox" checked="@selected.Contains(subject.Id)" value="@(subject.Id)" class="govuk-checkboxes__input">
-                          <label for="@("subject- " + subject.Id)" class="govuk-label govuk-checkboxes__label govuk-label--s">@subject.Name</label>
+                          <label for="@("subject- " + subject.Id)" class="govuk-label govuk-checkboxes__label">
+                            <span class="govuk-!-font-weight-bold govuk-!-margin-bottom-2">
+                              @subject.Name
+                            </span>
+                            @if (subject.Funding != null && subject.Funding.BursaryFirst.HasValue)
```

Comment:
The logic can be simplified here to one level: 
```
@{
string txt = null;
if (subject.Funding?.Scholarship != null) 
{
    txt = $"Scholarships and bursaries up to {String.Format("{0:C0}", subject.Funding.Scholarship.Value} available";
} 
else if (subject.Funding?.BursaryFirst != null)
{
   txt = $"Bursaries up to {String.Format("{0:C0}", subject.Funding.BursaryFirst.Value} available";
}
if (txt != null && subject.Funding?.EarlyCareerPayments != null)  
{
    txt += $" with further early career payments of {String.Format("{0:C0}", subject.Funding.EarlyCareerPayments.Value)}.";
}
else if (txt != null)
{
    txt += ".";
}
}
<span class="govuk-!-display-block">@txt</span>
```

reject

Reply:
I'd also consider moving the code block into the view model or a helper method

suggestion